app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/DataBindingIdlingResource.kt

```diff
@@ -101,12 +101,3 @@ fun DataBindingIdlingResource.monitorActivity(
         this.activity = it
     }
 }
-
-/**
- * Sets the fragment from a [FragmentScenario] to be used from [DataBindingIdlingResource].
- */
-fun DataBindingIdlingResource.monitorFragment(fragmentScenario: FragmentScenario<out Fragment>) {
```

Comment:
I prefer to keep this, this is more library code than sample code really.

Reply:
It wasn't compiling anymore, not sure why

Reply:
oh then ok